hungarian/README.md

```diff
@@ -0,0 +1,52 @@
+# Hungarian braille specifications, latest 2019 determined standard
+The official hungarian grade 1 and grade 2 braille standard is deterbined with The Hungarian Braille Committee official in 2012, and latest modification date is 2019. This subpage always containing latest official accepted hungarian braille standards for Liblouis braille developer community.
+The hungarian braille tables affected with this standard is following:
+* tables/hu-hu-comp8.ctb: the official accepted 8-dots computer braille table. In hungary we using only eight dot mode with computer braille format, other variations is six dot type (the full braille and the grade 2 tables). This is the official standard mode.
+* tables/hu-hu-g1.ctb: the official accepted grade 1 braille (six dot mode), this is the full braille mode table.
+* tables/hu-hu-g2.ctb: this table containing the hungarian grade 2 braille rules, with official uses hungari in literary books, hungarian braille news papers, etc. Hungarian grade 2 braille have two variations, small, fever abbreviations containing rule set (this variation containing only 45 abbreviation rules), and larger variations, with need using literary books, braille printing documents.
+In 2017, the larger variation is implemented.
+
+Official table maintainer from Liblouis community in Hungary: Attila Hammer from [IT Foundation for the Visually Impaired - Hungary](http://www.infoalap.hu).
+Attila is subscribed in Liblouis list. If anybody have question the hungarian braille tables related, feel free ask a quastion in developer community.
+Please ask beginner table maintainers:
+If would like doing bigger changes in hungarian braille tables related, please not commit the changes official the source [Liblouis repository](https://github.com/liblouis/liblouis) until all testcases are not passed, because hungarian language is a very complex language.
+If a maintenance change related have a question, have possibility to write e-mail a question the table maintainer with following addresses:
+* Full official work time used e-mail address, with awailable the hungarian braille tables: hammer.attila at infoalap dot hu
+* Private e-mail address, with Attila subscribed into Liblouis mailing list: hammera at pickup dot hu
```

Comment:
All this information is about the Liblouis tables, and about you, the maintainer of the tables. I think it would be better if this information would go in the tables themselves. With the "braille-specs" sub-project we try to provide general information that goes beyond the implementation.

Reply:
This issue is not resolved yet.